1.0/docs/start/psk/creating-a-page.md

```diff
@@ -0,0 +1,211 @@
+---
+layout: default
+type: start
+shortname: Start
+title: Creating a new page
+subtitle: Polymer Starter Kit
+---
+
+## Creating a new page
+
+This Polymer Starter Kit tutorial teaches you how to:
+
+* create a new menu item in the navigation menu
+* create content for a new page
+* route traffic to your new page
+
+This guide assumes you have already completed the [Set up guide](setting-up.html).
+
+## Setting up
+
+1. `cd` to the base directory of your project.
+
+1. Start the local development server.
+
+       grunt
+
+   The local development server opens the application in your default
+   web browser. As you modify your application, the server detects the
+   modifications, re-builds the application, and reloads your browser
+   automatically. There is no need to re-load the browser or the application.
+
+## Creating a navigation menu item
+
+1. Open `app/index.html`. 
+
+1. Find the navigation menu. The beginning of it is diplayed in the code
+   snippet below.
+
+{% highlight html %}
+...
+<!-- Drawer Content -->
+<paper-menu class="list" attr-for-selected="data-route" selected="{{route}}" on-iron-select="onMenuSelect">
+  <a data-route="home" href="/">
+    <iron-icon icon="home"></iron-icon>
+    <span>Home</span>
+  </a>
+...
+{% endhighlight %}
+
+Each navigation menu item consists of an `a` element with two children: 
+`iron-icon` and `span`.
```

Comment:
Our site style is to use angle brackets on element names, so `<iron-icon>`, `<span>`.